### ISSUE TITLE
4coder does not allow number from keypad

### ISSUE THREAD

**Issue by vaualbus** [4coder-editor/4coder#259]
Version: 4.1.5

**System**
OS: *Windows 10 1904*

I don't know if is my PC that enter some strange configuration state, but apparently last version of 4coder does not detect number written using the keypad. I didn't use the last version since it was released but today I begin to using it again and I found this issue. Been on windows it can easily be that is windows that decide to break some API during an update.

**Comment by mrmixer** [comment]
I believe this is fixed in 4.1.6, you should try downloading that version. Be careful that 4.1.6 doesn't use physical key positions for keybindings anymore (by default, there is a setting in config.4coder), so you might need to redo your bindings. This is the same issue as https://github.com/4coder-editor/4coder/issues/246

**Comment by vaualbus** [comment]
Yeah I didn't realized that there was an update of 4coder. Now indeed it work!
Sorry for the "bad" bug report
